backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -48,38 +48,36 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
     * If the `work` `Future` completes successfully, perform the `onSuccess` work, otherwise schedule
     * the execution of the `onFailure` work using an exponential backoff.
     */
-  def withRetry(work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit): Unit = {
+  def withRetry(actorContext: ActorContext, work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit)(implicit ec: ExecutionContext): Unit = {
```

Comment:
If I was personally going to rationalise all this async stuff, it wouldn't be by trying to crowbar it into yet-another-`retry`-method.

The whole retry logic here just seems philosophically anti-actor to me?